RELEASENOTES.md

```diff
@@ -2,6 +2,20 @@
 
 - Issue 1184 Publish to Environment fails on 'Permission Denied'
 
+### New Settings
+
+- `deliverTo<deliverytarget>` now has an additional property called `ContinuousDelivery`, indicating whether or not to run continuous delivery to this deliveryTarget. Default is true.
+- `trustMicrosoftNuGetFeeds` Unless this setting is set to false, AL-Go for GitHub will trust the NuGet feeds provided by Microsoft. The feeds provided by Microsoft contains all Microsoft apps, all Microsoft symbols and symbols for all AppSource apps.
+- `trustedNuGetFeeds` - can be an array of NuGet feed specifications, which AL-Go for GitHub will use for dependency resolution. Every feed specification must include a url property and can optionally include a few other properties:
+  - Url - The URL of the feed (examples: https://pkgs.dev.azure.com/myorg/apps/\_packaging/myrepo/nuget/v3/index.json or https://nuget.pkg.github.com/mygithuborg/index.json")
+  - Patterns - AL-Go for Github will only trust packages, where the ID matches this pattern. Default is all packages (\*).
+  - FingerPrints - If specified, AL-Go for GitHub will only trust packages signed with one of the fingerprints specified in this array of fingerprints.
```

Comment:
Wording could be better here: packages aren't _signed_ with fingerprints.

Reply:
I actually think it is called thumbprint all over - renaming before we ship.
Also requires a new BcContainerHelper in a sec.

Reply:
Corrected the dotnet tool for checking this uses fingerprints - I will stick with this implementation